Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -30,9 +30,11 @@ For more details about the authentication used in this integration, see [Microso
 * Machine.RestrictExecution - Application
 * Machine.Scan - Application
 * Machine.StopAndQuarantine - Application
-* ThreatIndicators.ReadWrite.OwnedBy - Application
+* ThreatIndicators.ReadWrite.OwnedBy - Application. Please note - this only used for the deprecated indicators command and it is not needed if not using them.
 * Url.Read.All - Application
 * User.Read.All - Application
+* Ti.ReadWrite (Read and write IOCs belonging to the app) - Application
```

Comment:
lets try to understand better if `Ti.ReadWrite` is sufficient and if we can remove `Ti.ReadWrite.All`

Reply:
It seems like after I removed `Ti.ReadWrite.All` all of the commands are still working.